src/serac/physics/utilities/functional/integral.hpp

```diff
@@ -557,8 +562,11 @@ class Integral {
            Dimension<spatial_dim>, lambda_type&& qf)
       : J_(J), X_(X)
   {
-    constexpr auto geometry = supported_geometries[geometry_dim];
-    constexpr auto Q        = std::max(test_space::order, trial_space::order) + 1;
+    constexpr auto geometry                      = supported_geometries[geometry_dim];
+    constexpr auto Q                             = std::max(test_space::order, trial_space::order) + 1;
+    constexpr auto quadrature_points_per_element = (spatial_dim == 2) ? Q * Q : Q * Q * Q;
+
+    uint32_t num_quadrature_points = quadrature_points_per_element * uint32_t(num_elements);
```

Comment:
Why are you using a fixed width type?

Reply:
I'm not sure what the appropriate type should be. Part of the weirdness is `mfem::Mesh::GetNE()` returning a 32-bit signed integer. (?!)

Would you prefer: `size_t`, `uint64_t`, or something else?

Reply:
Is there a guarantee this line is not gonna go past 32 bits after you multiply them together? `size_t` is generally 64 so it does give you quite a bit more buffer but I'm not familiar enough with how large these two things end up being

Reply:
There's definitely a chance of overflow on the multiplication, so it should be greater than 32 bits. It would be good to update the mfem routines to use a larger type too.